integration-tests/discovery-tests/gateway/src/main/resources/application.yaml

```diff
@@ -33,6 +33,15 @@ spring:
               args:
                 retries: 3
                 series: SERVER_ERROR
+        - id: order
+          uri: lb://order
+          predicates:
+            - Path=/order/**
+          filters:
+            - name: Retry
+              args:
+                retries: 3
+                series: SERVER_ERROR
```

Comment:
Why add this route?

Reply:
because GatewayGovernanceIT some /order/* invoke failed.